src/parser/content-parser.php

```diff
@@ -26,13 +26,29 @@ public function __construct( $block_registry = null ) {
 		$this->block_registry = $block_registry;
 	}
 
+	public function should_block_be_included($block, $block_name, $filter_options ) {
+		$is_block_included = true;
+
+		if ( ! empty( $filter_options[ 'include' ] ) ) {
+			$is_block_included = in_array( $block_name, $filter_options[ 'include' ] );
+		} else if ( ! empty( $filter_options[ 'exclude' ] ) ) {
+			$is_block_included = ! in_array( $block_name, $filter_options[ 'exclude' ] );
+		}
+
+		return apply_filters( 'vip_block_data_api__content_filter_block', $is_block_included, $block_name, $block);
```

Comment:
How about `vip_block_data_api__allow_block` for this filter name? I think that makes it clearer that returning `false` removes a block and `true` allows it through.

Reply:
Can you include the PHPDoc header from the README above this filter in code as well?